proto/ibc/applications/interchain_accounts/controller/v1/tx.proto

```diff
@@ -38,18 +37,14 @@ message MsgSendTx {
 
   string owner         = 1;
   string connection_id = 2 [(gogoproto.moretags) = "yaml:\"connection_id\""];
-  // Timeout height relative to the current block height.
-  // The timeout is disabled when set to 0.
-  ibc.core.client.v1.Height timeout_height = 3
-      [(gogoproto.moretags) = "yaml:\"timeout_height\"", (gogoproto.nullable) = false];
   // Timeout timestamp in absolute nanoseconds since unix epoch.
   // The timeout is disabled when set to 0.
```

Comment:
Looks like the documentation needs to be moved and updated.

I'd suggest:
```suggestion
  // Relative timeout timestamp provided will be added to the current block time during transaction execution.
  // The timeout timestamp must be non-zero. 
```
This will close #2267